frontend/src/utils/utils.js

```diff
@@ -14,4 +14,15 @@ export const isValidEmail = (email) => {
 	return re.test(email);
 };
 
-// const ms_delay = ms => new Promise(res => setTimeout(res, ms));
\ No newline at end of file
+/**
+ * A simple delay function to wait for xxx miliseconds
+ * @param ms miliseconds to sleep
+ * Source: https://stackoverflow.com/a/47480429
+ * Usage:
+ * import delayMsec from "../utils";
+ * const yourFunction = async () => {
+ *     await delayMsec(5000);
+ *     console.log("Waited 5s");
+ *};
+ */
+export const delayMsec = ms => new Promise(res => setTimeout(res, ms));
```

Comment:
Great that you put this into `utils.js`. I wonder, whether there would have been an existing function that you could have used instead.